### PR TITLE
Standardize ping endpoints

### PR DESCRIPTION
**Pull request by kgang**
* Try to fix health checks on AWS

### CODE REVIEW

File: webapp/api/server.py [hunk 1]
```diff
@@ -28,7 +28,7 @@
 
 dotenv.load_dotenv(override=True)
 
-UNAUTHENTICATED_PATHS = {'/docs', '/redoc', '/openapi.json', '/favicon.ico'}
+UNAUTHENTICATED_PATHS = {'/ping', '/docs', '/redoc', '/openapi.json', '/favicon.ico'}
```
Comment (MichaelDeng03) [inline webapp/api/server.py:31]: I don't understand why you are adding ping. 
auth-staging is a healthy service, but its health check path is `/`

Comment (kgang) [reply webapp/api/server.py:31]: Just to standardize it? [](edited-with-Axolo)

Comment (MichaelDeng03) [reply webapp/api/server.py:31]: do you intend on changing future health checks to /ping, and I should automate that process in our CI/CD? [](edited-with-Axolo)

Comment (kgang) [reply webapp/api/server.py:31]: yeah I was going to change all the health checks to `/ping` [](edited-with-Axolo)

Comment (kgang) [reply webapp/api/server.py:31]: webservices should always have a `/ping` endpoint that's unauthenticated and returns a 200 [](edited-with-Axolo)

**Review by MichaelDeng03 — APPROVED** [review]

**Pull request merged**